app/javascript/readingList/readingList.jsx

```diff
@@ -189,14 +186,14 @@ export class ReadingList extends Component {
       ''
     );
     return (
-      <div>
+      <section>
```

Comment:
Some `<div />` soup cleanup.

Reply:
(just a thought) we still get a warning in `axe` about the `<header>` not being contained within a landmark (because `section` doesn't become an aria landmark unless it has a label).

It's probably a bigger/later refactor, but I think the structure of the page should be more like:

![screenshot showing main content encompassing a moved h1 and all of the controls apart from the tag list which remains a nav outside the main](https://user-images.githubusercontent.com/20773163/108235490-c4956180-713d-11eb-96f7-d1975010a50b.png)

Reply:
Yeah. This kind of ties into #1153 a bit, but I can make that change here.